staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -208,6 +208,7 @@ func (f *fancyResponseWriterDelegator) Flush() {
 }
 
 func (f *fancyResponseWriterDelegator) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	f.processCode(101)
```

Comment:
please add comment: // fake an before protocol switch happens

And please use the net/http StatusSwitchingProtocol constant.